.github/ISSUE_TEMPLATE/general-issue.yml

```diff
@@ -0,0 +1,16 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: General Issue
+description: A blank issue for general purpose use.
+title: "[Topic] Title"
+body:
+- type: textarea
+  attributes:
+    label: Description
+    value: |
+      Give a description of the problem, listing similar issues and if able:
+      - Set a milestone for the issue.
```

Comment:
I suggest deleting this. Typically people reporting issues shouldn't assign a milestone IMO (unless they already know that they should).